api/resources/translations/messages-es.properties

```diff
@@ -379,6 +379,9 @@ branding.favicon.field =
 branding.logo.field = 
 branding.title.field = 
 branding.title.field.help =
+browser.compatibility.title = Contacte a su supervisor
+browser.compatibility.description = Por favor contacte a su supervisor para actualizar su navegador.
+browser.compatibility.confirm = Bueno
```

Comment:
Just want to make sure we are consistent with caps, do we normally use `Ok` or `OK` elsewhere in the app?

Reply:
@michaelkohn the existing `Ok` is the one for [Date confirmation](https://github.com/medic/cht-core/blob/6dec6344c60ca3c36ea267a475336a797a8b4172/api/resources/translations/messages-en.properties#L581) which is small caps. So that's what I used here to keep it consistent.